packages.yaml

```diff
@@ -363,6 +381,7 @@ packages:
       - hisat2@2.1.0
       - samtools@1.4
       - subread@1.5.2
+      - sra-tools@2.8.2-5
```

Comment:
We are going to install `sra-tools` as core, and compile it with the system compiler. Just remove this line. I'll take care of adding it right now and deploy.